### PR TITLE
Feature/as 1157 spoc and inspectors

### PR DESCRIPTION
**Pull request by lvnhmd**
Please do not merge as I probably need to show the **already** assigned inspectors and spocs in the sidebar

### CODE REVIEW

File: pages/establishment/dashboard/views/index.jsx [hunk 1]
```diff
@@ -30,33 +30,49 @@ const Index = ({
...
+            {
+              pelh && <ProfileLink type="pelh" profile={pelh} />
+            }
+            {
+              nprc && <ProfileLink type="nprc" profile={nprc} />
+            }
+          </dl>
+        </Sidebar>
+        <Sidebar>
+          <dl>
```
Comment (lennym) [inline pages/establishment/dashboard/views/index.jsx]: Not sure there should be two sidebars and two `<dl>`s here. Can we remove the 4 lines above here?

File: pages/establishment/dashboard/views/index.jsx [hunk 2]
```diff
@@ -30,33 +30,49 @@ const Index = ({
...
-    </div>
-  </Fragment>
-);
+    </Fragment>
+  )
+  ;
+};
 
-const mapStateToProps = ({ static: { establishment, allowedActions } }) => ({ establishment, allowedActions });
+const mapStateToProps = ({ static: { establishment, allowedActions, profile } }) => ({ establishment, allowedActions, profile });
```
Comment (lennym) [inline pages/establishment/dashboard/views/index.jsx]: If the only property you need from the profile is `asruAdmin` then it makes sense to only pass that.
```suggestion
const mapStateToProps = ({ static: { establishment, allowedActions, profile } }) => ({ establishment, allowedActions, asruAdmin: profile.asruAdmin });
```

File: pages/establishment/dashboard/views/index.jsx [hunk 3]
```diff
@@ -30,33 +30,49 @@ const Index = ({
...
-    <Header title={name} />
-    <div className="govuk-grid-row">
-      <div className="govuk-grid-column-two-thirds">
-        <PanelList
-          panels={links.filter(link => allowedActions.includes(link.permissions)).map(link => <DashboardLink key={link.path} { ...link } />)}
-        />
+  allowedActions,
+  profile: {
+    asruAdmin
+  }
```
Comment (lennym) [inline pages/establishment/dashboard/views/index.jsx]: Assuming you take my suggestion below, then `asruAdmin` would be a top-level prop.
```suggestion
  allowedActions,
  asruAdmin
```

File: pages/establishment/dashboard/views/index.jsx (hunk 1 above)
Comment (lvnhmd) [reply pages/establishment/dashboard/views/index.jsx]: done

File: pages/establishment/dashboard/views/index.jsx (hunk 2 above)
Comment (lvnhmd) [reply pages/establishment/dashboard/views/index.jsx]: done

File: pages/establishment/dashboard/views/index.jsx (hunk 3 above)
Comment (lvnhmd) [reply pages/establishment/dashboard/views/index.jsx]: done

File: pages/establishment/dashboard/views/index.jsx [hunk 4]
```diff
@@ -28,35 +28,66 @@ const Index = ({
...
-    </div>
-  </Fragment>
-);
+    </Fragment>
+  )
+  ;
+};
 
-const mapStateToProps = ({ static: { establishment, allowedActions } }) => ({ establishment, allowedActions });
+const mapStateToProps = ({ static: { establishment, allowedActions, profile } }) => ({ establishment, allowedActions, asruAdmin: profile.asruAdmin });
```
Comment (lennym) [inline pages/establishment/dashboard/views/index.jsx]: I realised this needs to also check `asruUser`
```suggestion
const mapStateToProps = ({ static: { establishment, allowedActions, profile } }) => ({ establishment, allowedActions, asruAdmin: profile.asruUser && profile.asruAdmin });
```

Comment (lvnhmd) [reply pages/establishment/dashboard/views/index.jsx]: done

**Pull request merged**